src/sdp/DefaultSDP.ts

```diff
@@ -619,4 +643,36 @@ export default class DefaultSDP implements SDP {
     const newSDP = sections.join('');
     return new DefaultSDP(newSDP);
   }
+
+  mediaSections(): SDPMediaSection[] {
+    const sections = DefaultSDP.splitSections(this.sdp);
+    if (sections.length < 2) {
+      return [];
+    }
+
+    const parsedSections: SDPMediaSection[] = [];
```

Comment:
nit: Feel like the code is easier to read if we name this parsedMediaSections :p